src/users/factories.py

```diff
@@ -24,6 +24,5 @@ class MentorFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = models.Mentor
     profile = factory.SubFactory(ProfileFactory)
-    major = factory.SubFactory(MajorFactory)
```

Comment:
I took this out because every time a new mentor was created via factory, it would create a duplicate empty major, which was throwing off the tests that were checking to see if duplicate majors were being created